src/main/java/com/stripe/model/ThreeDSecure.java

```diff
@@ -0,0 +1,114 @@
+package com.stripe.model;
+
+import com.google.gson.annotations.SerializedName;
+
+import com.stripe.Stripe;
+import com.stripe.exception.APIConnectionException;
+import com.stripe.exception.APIException;
+import com.stripe.exception.AuthenticationException;
+import com.stripe.exception.CardException;
+import com.stripe.exception.InvalidRequestException;
+import com.stripe.net.APIResource;
+import com.stripe.net.RequestOptions;
+
+import java.util.Map;
+
+public class ThreeDSecure extends APIResource implements HasId {
+  String id;
```

Comment:
Sorry for the nit, but I think the rest of the library uses tabs for indentation instead of the two spaces convention. Would you mind changing over this file and its corresponding test file?